### DISCUSSION TITLE
What are the differences between the different formats?

### DISCUSSION THREAD

**Discussion by sojuo** [InfinityLoop1308/PipePipe#491] [Q&A]
What is the difference between these different formats? What is the impact on storage space and traffic consumption?
![Audio](https://files.catbox.moe/alltvy.avif)
![Vedio](https://files.catbox.moe/1vsk37.avif)

**Comment by InfinityLoop1308** [comment]
WebM generally has higher quality and use more data comparing to m4a/mp4. 3gp is no longer used.

**Reply by sojuo** [reply]
> WebM generally has higher quality and use more data comparing to m4a/mp4. 3gp is no longer used.

That is, WebM consumes the most resources when transferring exactly the same screen?
Also which one consumes less resources, MPEG-4 or 3GP, when transferring the same content?
